backend/core/src/main/resources/db/migration/V12__tags_add_source.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tags
```

Comment:
Merge all migrations into 1

Reply:
It would be hard to navigate through 1 file containing all the differences, is it worth it?

Reply:
This is a snapshot not a code to change/update.

Reply:
Ok

Reply:
Done, all merged